Content.Shared/_CD/Storage/EntitySystems/SharedMouthStorageSystem.cs

```diff
@@ -0,0 +1,84 @@
+using Content.Shared.Actions;
+using Content.Shared.CombatMode;
+using Content.Shared.Damage;
+using Content.Shared.CD.Storage.Components;
+using Content.Shared.Examine;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Standing;
+using Content.Shared.Storage;
+using Content.Shared.Storage.EntitySystems;
+using Robust.Shared.Containers;
+using Robust.Shared.Map;
+
+namespace Content.Shared.CD.Storage.EntitySystems;
+
+public abstract class SharedMouthStorageSystem : EntitySystem
+{
+    [Dependency] private readonly DumpableSystem _dumpableSystem = default!;
+    [Dependency] private readonly SharedContainerSystem _containerSystem = default!;
+    [Dependency] private readonly SharedActionsSystem _actionsSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<MouthStorageComponent, MapInitEvent>(OnMouthStorageInit);
```

Comment:
Did I miss something? Or is this only initialized on map init? Does it initialize properly when you spawn in a Urist with the f5 menu? If so, is this needed?

Reply:
It initializes properly on spawned Urists, and removing it causes the mouth storage action to completely disappear. It's needed as far as I can tell.

Reply:
Should be fine then, I would guess that MapInitEvent gets called more often then I thought based on the documentation (I did not open rider to check).